contrib/internal/namingschematest/memcached.go

```diff
@@ -0,0 +1,28 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+
+package namingschematest
+
+import (
+	"testing"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/mocktracer"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// NewMemcachedOpNameTest generates a new test for memcached span operation names using the naming schema versioning.
+func NewMemcachedOpNameTest(genSpans GenSpansFn) func(t *testing.T) {
```

Comment:
Is this function only used in exactly one place?

Reply:
yes! my intention with this was to create a generic test for memcached integrations, but since we only have one it makes sense to move it there. Thanks for pointing out!